src/Linter/LintingException.php

```diff
@@ -16,6 +16,8 @@
 
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @final
```

Comment:
let's add TODO to make it real final on next major release

Reply:
done :+1: